index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <h1 id="title"> freeCodeCamp Survey form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action="" id="survey-form">
+      <div class="div-form">
+        <label for="name" id="name-label">Name</label>
+        <input type="text" id="name" placeholder="Enter the name" required />
+      </div>
```

Comment:
Please also add a descriptive value for action, the attribute should not be empty. This makes your code unclean